infrastructure/export.py

```diff
@@ -241,18 +267,33 @@ def refresh_views():
                         ELSE 'NULL'
                     END
                     project_capacity_unit,
+                    project_capacity_timeframe,
                     project_output,
                     CASE
                         {project_output_unit}
                         ELSE 'NULL'
                     END
                     project_output_unit,
                     project_output_year,
-                    sox_reduction_system
+                    sox_reduction_system,
+                    linear_length AS "project_length",
+                    pipeline_diameter,
+                    pipeline_diameter_unit,
+                    upper(pipeline_metered::text) AS pipeline_metered,
```

Comment:
Neat way to get pretty boolean output in SQL.

After thinking about this in context of the existing export, this will behave differently than the other boolean fields though, like ``sox_reduction_system``, ``direct_current``, etc. My gut says to leave this as a raw value unless we want to update all boolean values, otherwise the analysis they do will have to use specific logic for this field.